meson.build

```diff
@@ -2499,9 +2496,7 @@ foreach dict : executables
         if is_fuzz
                 fuzzer_exes += exe
 
-                if want_tests != 'false'
-                        # Run the fuzz regression tests without any sanitizers enabled.
-                        # Additional invocations with sanitizers may get added below.
+                if want_tests != 'false' and want_fuzz_tests
```

Comment:
`fuzz-tests` is `false` by default as far as I can see so apart from removing the nested sanitized build this PR also disables the unit tests. I think it would be better to flip `fuzz-tests` (and change its description because it still says that it runs the fuzz targets with ASan/UBSan).

Reply:
Ah nvm it's only instrumented when llvm-fuzz or oss-fuzz options are enabled as well. Should be fine to enable by default then, will give it a try and update the PR